src/components/InCall.tsx

```diff
@@ -38,26 +47,40 @@ export const InCall = ({
 }: Props) => {
   const divRef = useRef(null);
   const [jitsiMeet, setJitsiMeet] = useState<IJitsiMeetApi>();
+  const [transcript, setTranscript] = useState<string>("");
+  const transcriptManager = useRef(new TranscriptManager(setTranscript));
```

Comment:
nit: you never seem to change this `ref`, so this can use the simpler `useState` or just `useMemo`.